src/Integration.Vsix.UnitTests/CFamily/IntegrationTests/CFamily_CLangAnalyzer_IntegrationTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using SonarLint.VisualStudio.Integration.Vsix.CFamily;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests.CFamily.IntegrationTests
+{
+    [TestClass]
+    public class CFamily_CLangAnalyzer_IntegrationTests
+    {
+        private string testsDataDirectory;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            testsDataDirectory = Path.Combine(Environment.CurrentDirectory, "CFamily\\IntegrationTests\\");
```

Comment:
The other tests that create directories create them under `TestContext.DeploymentDirectory`. I thought there was a helper function that did this, but I can't find it. Most of the tests use a method like this:

```
        private string CreateTestSpecificDirectory()
        {
            var dir = Path.Combine(TestContext.DeploymentDirectory, TestContext.TestName);
            Directory.CreateDirectory(dir);
            return dir;
        }
```